### DISCUSSION TITLE
ASUS VivoMini VC65-C1 + Ubuntu 24.04 LTS

### DISCUSSION THREAD

**Discussion by gnubit-au** [OpenSurv/OpenSurv#13] [Hardware + Operating system]
[https://www.asus.com/au/commercial-desktop/vivomini-vc65-c1/](url)
Hardware specs: i58400T ( Intel® UHD Graphics 630)
8GB RAM
Single 1080p Monitor.
OS: xubuntu-24.04.1-minimal-amd64
25 RTSP streams (640x480 @15fps) via a mediamtx media proxy server [https://github.com/bluenviron/mediamtx](url)
![btop_vc65](https://github.com/user-attachments/assets/b42fe640-33b1-4eb6-be32-bc088ec88b71)

**Comment by SvenVD** [comment]
Many thanks for sharing! This will help others looking for hardware that is known to be working.
